quattor/types/aii.pan

```diff
@@ -30,9 +30,17 @@ type structure_aii_hooklist = {
     "status" ? structure_aii_hook[]
 };
 
+@documentation{
+    AII structure which defines the osinstall, nbp,
+    dhcp config and hooks.
+    The "protected" value can be used to protect the install and removal.
+    By setting this value (eg. to the latest buildid), you will be asked 
+    to repeat this string as verification when trying to do install, configure or removal actions. 
```

Comment:
"repeat this string" -> expand this - where do I have to specify it? I think it's the --confirmation argument to aii-shellfe, correct?